doc/modules/generation.rst

```diff
@@ -0,0 +1,12 @@
+Generation module
+=================
```

Comment:
```suggestion
Generation module (in dev)
==========================
```
Will this be ready or is this a place holder? It seems to me that we should document that this is in development and not fully ready yet since it is a place holding module. But maybe this is a very temporary place holder with a very big code infusion coming soon?